analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -94,6 +98,19 @@ def parse_clang_help_page(
     return res
 
 
+def _is_user_disabled_checker(checker, ordered_checkers):
+    """
+    This function returns True if the given statistics-based checker is
```

Comment:
Seems like this is not a statisticsbased specific function. If it is important for those checkers specifically, document it why. I'd expect to see something like

"Since these checkers can be enabled through -e/--enable AND --stats, we need special handling for them to make sure -d/--disable overwrite --stats"